library/boehm_shim/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "boehm_shim"
+version = "0.1.0"
+authors = ["Jacob Hughes <jh@jakehughes.uk>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
Not sure we need this line?

Reply:
Good catch. Copy / paste autopilot

Reply:
Fixed: f055f76525faed922ad03743d6c337287e22bea7